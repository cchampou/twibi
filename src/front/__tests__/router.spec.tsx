import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Router from '../router';

describe('Router', () => {
  it('should contain a dashboard route', () => {
    expect(render(<Router />)).toMatchSnapshot();
  });
});
