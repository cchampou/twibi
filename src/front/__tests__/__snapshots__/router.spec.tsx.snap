// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Router should contain a dashboard route 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="css-1a1vh9v"
      >
        <img
          alt="Twibi logo"
          class="css-73mmas"
          size="1"
          src="a random file"
        />
        <h1
          class="css-gcnqix"
        >
          Twibi
        </h1>
        <h1
          class="css-1fy1jtn"
        >
          Twitch & social networks tools
        </h1>
        <a
          href="https://id.twitch.tv/oauth2/authorize?client_id=undefined&redirect_uri=undefined&response_type=token&scope=user:read:email chat:read chat:edit channel:read:subscriptions"
          target="_self"
        >
          <button
            class="css-1q1ryt5"
            type="button"
          >
            Enter
          </button>
        </a>
        <img
          alt="down"
          class="css-9mop3v"
          size="2"
          src="a random file"
        />
      </div>
      <div
        class="css-k164mj"
      >
        <div
          class="css-1488l36"
        >
          <h1
            class="css-1fy1jtn"
          >
            Twitch chat notifications
          </h1>
          <p
            class="css-6w30xp"
          >
            Get informed when a viewer is following, hosting, subscribing, and more !
          </p>
        </div>
        <div
          class="css-1488l36"
        >
          <img
            alt="twitch"
            class="css-146q2af"
            size="2"
            src="a random file"
          />
        </div>
      </div>
      <div
        class="css-wt80aq"
      >
        <div
          class="css-1488l36"
        >
          <img
            alt="discord"
            class="css-146q2af"
            size="2"
            src="a random file"
          />
        </div>
        <div
          class="css-1488l36"
        >
          <h1
            class="css-1fy1jtn"
          >
            Now live notifications
          </h1>
          <p
            class="css-6w30xp"
          >
            Inform your fans you are live on your favorite social networks, like Discord, Twitter & more.
          </p>
        </div>
      </div>
      <footer
        class="css-1o89dal"
      >
        <a
          class="css-9srvqk"
          href="https://www.github.com/cchampou/twibi"
          target="_blank"
        >
          <svg
            aria-hidden="true"
            class="svg-inline--fa fa-github fa-w-16 "
            data-icon="github"
            data-prefix="fab"
            focusable="false"
            role="img"
            viewBox="0 0 496 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"
              fill="currentColor"
            />
          </svg>
           Github 
          <svg
            aria-hidden="true"
            class="svg-inline--fa fa-external-link-alt fa-w-16 "
            data-icon="external-link-alt"
            data-prefix="fas"
            focusable="false"
            role="img"
            viewBox="0 0 512 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z"
              fill="currentColor"
            />
          </svg>
        </a>
      </footer>
    </div>
  </body>,
  "container": <div>
    <div
      class="css-1a1vh9v"
    >
      <img
        alt="Twibi logo"
        class="css-73mmas"
        size="1"
        src="a random file"
      />
      <h1
        class="css-gcnqix"
      >
        Twibi
      </h1>
      <h1
        class="css-1fy1jtn"
      >
        Twitch & social networks tools
      </h1>
      <a
        href="https://id.twitch.tv/oauth2/authorize?client_id=undefined&redirect_uri=undefined&response_type=token&scope=user:read:email chat:read chat:edit channel:read:subscriptions"
        target="_self"
      >
        <button
          class="css-1q1ryt5"
          type="button"
        >
          Enter
        </button>
      </a>
      <img
        alt="down"
        class="css-9mop3v"
        size="2"
        src="a random file"
      />
    </div>
    <div
      class="css-k164mj"
    >
      <div
        class="css-1488l36"
      >
        <h1
          class="css-1fy1jtn"
        >
          Twitch chat notifications
        </h1>
        <p
          class="css-6w30xp"
        >
          Get informed when a viewer is following, hosting, subscribing, and more !
        </p>
      </div>
      <div
        class="css-1488l36"
      >
        <img
          alt="twitch"
          class="css-146q2af"
          size="2"
          src="a random file"
        />
      </div>
    </div>
    <div
      class="css-wt80aq"
    >
      <div
        class="css-1488l36"
      >
        <img
          alt="discord"
          class="css-146q2af"
          size="2"
          src="a random file"
        />
      </div>
      <div
        class="css-1488l36"
      >
        <h1
          class="css-1fy1jtn"
        >
          Now live notifications
        </h1>
        <p
          class="css-6w30xp"
        >
          Inform your fans you are live on your favorite social networks, like Discord, Twitter & more.
        </p>
      </div>
    </div>
    <footer
      class="css-1o89dal"
    >
      <a
        class="css-9srvqk"
        href="https://www.github.com/cchampou/twibi"
        target="_blank"
      >
        <svg
          aria-hidden="true"
          class="svg-inline--fa fa-github fa-w-16 "
          data-icon="github"
          data-prefix="fab"
          focusable="false"
          role="img"
          viewBox="0 0 496 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"
            fill="currentColor"
          />
        </svg>
         Github 
        <svg
          aria-hidden="true"
          class="svg-inline--fa fa-external-link-alt fa-w-16 "
          data-icon="external-link-alt"
          data-prefix="fas"
          focusable="false"
          role="img"
          viewBox="0 0 512 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z"
            fill="currentColor"
          />
        </svg>
      </a>
    </footer>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
